packages/js/client/src/__tests__/Web3ApiClient.spec.ts

```diff
@@ -2458,6 +2458,7 @@ scalar Int32
 scalar Bytes
 scalar BigInt
 scalar JSON
```

Comment:
This file looks like it was left over from the Map PR by accident, is that correct?

Reply:
I believe it's because of the way we build plugins. We use the latest released version of the toolchain to build the plugin that's why some changes are only reflected after the release.

Reply:
Ah, ok